### ISSUE TITLE
[Discussion] Remove "domain" property?

### ISSUE THREAD

**Issue by Nightfirecat** [jdm-contrib/justdelete.me#181]
As-is, while we do have repositories for the [Chrome](https://github.com/jdm-contrib/justdelete.me-chrome-extension) and [Firefox](https://github.com/jdm-contrib/justdelete.me-firefox-addon), we do not actively maintain them. Neither seems to have an active userbase (15603 users of the Chrome extension, and 27 users of the Firefox addon, based on the respective sites' stats) and neither have active comments/feedback, leading me to believe that they would not be missed if they were removed.

This would help remove a fair number of extra "domain" property entries in sites.json that would no longer be needed.

**Comment by tupaschoal** [comment]
Hmm, I have mixed feelings. It doesn't really cost us much keeping the domains on the JSON, on the other hand, the apps have not been updated for a few years.

**Comment by Nightfirecat** [comment]
I agree that the cost is relatively low given that this property doesn't (currently) affect how the web page is rendered. I just see it as an easy target for removal given that its only purpose is for a pair of webextensions (for which, the Firefox extension doesn't even run in the latest version!) that we aren't maintaining.

**Comment by tupaschoal** [comment]
So I'll revert the thought a bit. How difficult would it be to get the chrome extension up and running again, do you know? Can we deploy to the webstore easily? 

Because I find the extensions useful, though I do not use them, and it would be great for diverging the traffic from the old repo to our maintained one, aside from making some publicity. 15k users is not that much if you think about the whole community of chrome users, but it is not dismissible either, given our niche application.

**Comment by green-leader** [comment]
I just installed the chrome extension and it actually appears to be working,

**Comment by green-leader** [comment]
I just looked and it actually appears as if the chrome extension is currently working and on the webstore.
